Assets/Scripts/EnemyController.cs

```diff
@@ -309,7 +309,13 @@ public void OnDamageTaken()
         Debug.Log("Enemy took damage. Current health: " + currentHealth);
         audioSourceHit.volume = volumeHit * sfxVolume;
         audioSourceHit.Play();
-        HurtSound();
+
+        int r = Random.Range(0,10);
+        if (r < 2)
+        {
+            HurtSound();
+        }
+
```

Comment:
Why do we do that? Just interested

Reply:
also 
```suggestion
        if (Random.Range(0,10) < 2)
        {
            HurtSound();
        }

```

Reply:
It makes more sense to me that there's a chance to run the sound function, rather that running the sound function has a chance to even play the sound.